capture-sdk/sdk/src/main/java/net/gini/android/capture/internal/camera/view/QRCodePopup.kt

```diff
@@ -113,7 +115,9 @@ internal class QRCodePopup<T> @JvmOverloads constructor(
                     R.color.Success_05
                 )
             )
+            performHapticFeedback(HapticFeedbackConstants.CONFIRM)
```

Comment:
[Clarification] The Android documentation says the `CONFIRM` and `REJECT` constants are available from [api level 30 (Android 11)](https://developer.android.com/reference/kotlin/android/view/HapticFeedbackConstants?hl=en#CONFIRM:kotlin.Int). Do you have a device running Android 10 or older to test it? I will test it as well, but my device is still charging.